modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -7,32 +7,68 @@
 = Using Data Mover for CSI snapshots
 
 :FeatureName: Data Mover for CSI snapshots
-include::snippets/technology-preview.adoc[]
 
-The OADP 1.1.2 Data Mover enables customers to back up container storage interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications from the store if a failure, accidental deletion, or corruption of the cluster occurs. The OADP 1.1.2 Data Mover solution uses the Restic option of VolSync.
+The OADP Data Mover enables customers to back up Container Storage Interface (CSI) volume snapshots to a remote object store. When Data Mover is enabled, you can restore stateful applications, using CSI volume snapshots pulled from the object store if a failure, accidental deletion, or corruption of the cluster occurs.
+
+The Data Mover solution uses the Restic option of VolSync.
 
-[NOTE]
-=====
 Data Mover supports backup and restore of CSI volume snapshots only.
 
-Currently, Data Mover does not support Google Cloud Storage (GCS) buckets.
-=====
+In OADP 1.2.x Data Mover `VolumeSnapshotBackups` (VSBs) and `VolumeSnapshotRestores` (VSRs) are queued using the VolumeSnapshotMover (VSM). The VSM's performance is improved by specifying a concurrent number of VSBs and VSRs simultaneously `InProgress`. After all async plugin operations are completed, the backup is marked as completed.
+
+
+[NOTE]
+====
+The OADP 1.2.x Data Mover is a fully supported Red Hat feature.
+
+The OADP 1.1.x Data Mover is a Technology Preview feature.
+====
+:FeatureName: The OADP 1.1.x Data Mover
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[NOTE]
+====
+Red Hat recommends that customers who use OADP 1.2.x Data Mover to back up and restore ODF CephFS volumes, upgrade or install OCP 4.12 for improved performance. OADP Data Mover can leverage CephFS shallow volumes in OCP 4.12+, which based on our testing improves the performance of backup times.
```

Comment:
I find this sentence awkward. I first read it that upgrade or install was something you could do with Data Mover. At the very least s/OCP/{product-version}

> For customers who use OADP 1.2.x Data Mover to back up and restore ODF CephFS volumes, Red Hat recommends that you upgrade to or install {product-version} 4.12 for improved performance

Reply:
Should the recommendation be for 4.12 **_or greater_**?

Reply:
> which based on our testing improves the performance of backup times.

This makes me a bit uncomfortable, making promises related to a 3rd party product. 

If you leave it, can you tighten it up a bit?

>which, based on our testing, can improve~~s the performance of~~ backup times.

Reply:
> can leverage CephFS shallow volumes in OCP 4.12+

Can they not leverage CephFS shallow volumes in OCP 4.11, or are they not available in 4.11?